src/main/java/model/Database/Queries.java

```diff
@@ -7,21 +7,40 @@
 
     public static class Create {
 
-        public static String CREATE_TABLE_RESERVATIONS = ""; // TODO: Replace empty string with query
-
-        public static String CREATE_TABLE_RENTALS = ""; // TODO: Replace empty string with query
-
-        public static String CREATE_TABLE_VEHICLES = ""; // TODO: Replace empty string with query
-
-        public static String CREATE_TABLE_VEHICLE_TYPES = ""; // TODO: Replace empty string with query
-
-        public static String CREATE_TABLE_CUSTOMERS = ""; // TODO: Replace empty string with query
-
-        public static String CREATE_TABLE_RETURNS = ""; // TODO: Replace empty string with query
-
-        public static String CREATE_TABLE_CREDIT_CARDS = ""; // TODO: Replace empty string with query
-
-        public static String CHECK_TABLE_EXISTS = ""; // TODO: Replace empty string with query
+        public static String CREATE_TABLE_RESERVATIONS = "CREATE TABLE IF NOT EXISTS Reservations(confNo INT, " +
```

Comment:
Maybe put each attribute on a newline for easier reading, like:
```java
public static String CREATE_TABLE_RESERVATIONS = 
    "CREATE TABLE IF NOT EXISTS Reservations(" +
        "confNo INT, " +
        "vtName CHAR(15) NOT NULL," + 
        "dLicense INT NOT NULL," +
        "fromDateTime TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP," +
        "toDateTime TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP," +
        "PRIMARY KEY (confNo), " +
        "FOREIGN KEY (vtName) REFERENCES VehicleType(vtName) ON DELETE CASCADE," + 
        "FOREIGN KEY (dLicense) REFERENCES Customer(dLicense) ON DELETE CASCADE" +
    ");";
```
This is mostly important because the TAs will be looking at this part of the code for 50% of the entire deliverable's grade, so we want to make it as clear and easy to read as possible. This is why I designed the framework such that all SQL queries are together in one file, separate from the rest of the code

Reply:
Also, I think having a default timestamp is a bad idea. If the `fromDateTime` or `toDateTime` is not given in the SQL statement, then we should be throwing an error in the program (and hopefully catch the bug and fix it in testing). Otherwise we might not notice the bug and the database can end up with weird values where the `fromDateTime` is after the `toDateTime`

Further, let's make `vtName` longer than 15 characters. We don't have any shortage of space on our DB, and a lower char count is more prone to errors. Maybe make it like 50

Reply:
Same comments for all other SQL statements below

Reply:
> Also, I think having a default timestamp is a bad idea. If the `fromDateTime` or `toDateTime` is not given in the SQL statement, then we should be throwing an error in the program (and hopefully catch the bug and fix it in testing). Otherwise we might not notice the bug and the database can end up with weird values where the `fromDateTime` is after the `toDateTime`
> 
> Further, let's make `vtName` longer than 15 characters. We don't have any shortage of space on our DB, and a lower char count is more prone to errors. Maybe make it like 50
 The main reason I set a default value is because the default value that MySQL sets is not compatible with TimeStamp.

Reply:
ah ok, maybe we just have to be careful with it